### ISSUE TITLE
BKG - Backgammon game

### ISSUE THREAD

**Issue by atsampson** [PDP-10/its#791]
The list of games in JOBS includes BKG and .BACKG. There's no TS BKG or similar in the AI/MC public tarballs.

However, SRI-NIC has SRC:<GAMES>BKG.EXE, which says "This is C-MU's BKG V9.9, Alias Mighty-B". This is Hans Berliner's implementation, originally written in BLISS for TOPS-10:

http://repository.cmu.edu/cgi/viewcontent.cgi?article=3408&context=compsci
http://www.bkgm.com/articles/Berliner/
https://www.cmu.edu/news/stories/archives/2017/january/obituary-berliner.html

Was ITS's game the same thing, wrapped with DECUUO?

**Comment by eswenson1** [comment]
Binary installed as part of DB-ITS. Still need to build from source.  #907

**Comment by atsampson** [comment]
TS BKG is CMU BKG V9.8 with a DECUUO bootstrap.

**Comment by eswenson1** [comment]
Do you have source?

**Comment by atsampson** [comment]
Nope. I don't know if it was ever distributed as source; all I could find when looking before was the BKG.EXE binary.
